you-should-use.plugin.zsh

```diff
@@ -2,22 +2,16 @@
 
 export YSU_VERSION='1.8.0'
 
-if ! type "tput" > /dev/null; then
-    printf "WARNING: tput command not found on your PATH.\n"
-    printf "zsh-you-should-use will fallback to uncoloured messages\n"
-else
-    NONE="$(tput sgr0)"
-    BOLD="$(tput bold)"
-    RED="$(tput setaf 1)"
-    YELLOW="$(tput setaf 3)"
-    PURPLE="$(tput setaf 5)"
-fi
+NONE="\e[0m"
+BOLD="\e[1m"
+RED="\e[31m"
+YELLOW="\e[33m"
+PURPLE="\e[35m"
```

Comment:
so unfortunately - using tput required for ensuring compatibility with all terminals. Happy that you removed `wc` though :)